charts/coturn/templates/secret.yaml

```diff
@@ -7,6 +12,8 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 type: Opaque
-data:
-  zrest_secret.txt: {{ .Values.secrets.zrestSecret | b64enc | quote }}
-
+stringData:
+  zrest_secret.txt: |
+    {{- range .Values.secrets.zrestSecrets }}{{ . | nindent 4 }}
```

Comment:
Could you please confirm that `b64enc` (base64 encoding) isn't needed for values of the list?

(It looks a bit suspicious, but might be fine.)

Reply:
Yes, I'm purposefully using `stringData` here instead of `data` -- see [this API spec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#secret-v1-core).

Reply:
Didn't know that one :smiley_cat: